backend/lib/endpoints/posts.js

```diff
@@ -332,7 +332,7 @@ async function routes(app) {
       schema: updatePostSchema,
     },
     async (req) => {
-      const { userId } = req;
+      const { userId, body: postProps } = req;
```

Comment:
one line is probably not sufficient
in the code below there's also 
`const { body } = req;`
and then both `postProps` and `body` are used so should be one const

Reply:
Silly me just fixing error without looking at rest of code. :sweat_smile: 

The `{ body: postProps }` thing is only used in POST post. Other places use `{ body } = req` or just `req.body` so we're not exactly consistent. I think `{ body } = req` is good, you?